sources/packages/web/src/components/generic/formio.vue

```diff
@@ -201,7 +177,13 @@ export default defineComponent({
       form.on("render", (event: HTMLElement) => {
         context.emit("render", form, event);
       });
-      isFormCreated.value = true;
+
+      // When the form submission data is set, the form is rendered again.
+      // To visually hide the form until the rendering to be completed, a delay is set.
+      setTimeout(() => {
+        isFormCreated.value = true;
+        context.emit("loaded", form);
+      }, 200);
```

Comment:
Can we have the `200` created as a const?
Suggestion: `FORMIO_LOAD_DATA_PROCESSING_VIEW_DELAY`.